.github/workflows/acceptance-tests.yml

```diff
@@ -1,31 +1,58 @@
 name: Acceptance Tests
 
 on:
-  schedule:
-    - cron: "30 7 * * *"
-  workflow_dispatch:
+  issue_comment:
+    types: [created]
+
 
 concurrency:
   group: ci-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
   acceptance_tests:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Any reason to go from a LTS to a latest?

Reply:
no good reason, i was blindly updating to latest, will revert this, thanks.

Reply:
Probably you can use the latest LTS

Reply:
runs on latest lts ( 20.04 ) now! thanks